src/Pim/Component/Catalog/Normalizer/Indexing/Product/CompletenessesNormalizer.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Pim\Component\Catalog\Normalizer\Indexing\Product;
+
+use Doctrine\Common\Collections\Collection;
+use Pim\Component\Catalog\Model\CompletenessInterface;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+
+/**
+ * Normalize the completenesses to the indexing format.
+ *
+ * @author    Julien Janvier <jjanvier@akeneo.com>
+ * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class CompletenessesNormalizer implements NormalizerInterface
```

Comment:
(not in your PR just an idea I have to introduce the word: "CompletenessCollection" to be consistent with the ProductValueCollection)